src/components/app.tsx

```diff
@@ -226,10 +232,12 @@ export class App extends React.PureComponent<IProps, IState> {
       }
 
       const showThemeButtons = queryValueBoolean("themeButtons");
+      // Show a warning about obsolute features if activity/sequences is marked as defunct
+      const showDefunctBanner = activity.defunct || (sequence?.defunct);
```

Comment:
Just a style thing but the parens about the sequence defunct check are not needed.